src/detokenization.rs

```diff
@@ -0,0 +1,50 @@
+use crate::Task;
+use serde::{Deserialize, Serialize};
+
+/// Input for a [crate::Client::detokenize] request.
+pub struct TaskDetokenization<'a> {
+    /// List of token ids which should be detokenized into text.
+    pub token_ids: &'a Vec<u32>,
```

Comment:
Please make this a slice `&'a [u32]`. There is now reason to demand the ids are on the heap if we only borrow the structure.

Reply:
If you borrow one thing, you might as well borrow everything. There is an argument to be made for always having an owned representation available via 'static. You can efficiently implement this via Cow